kubernetes/annotations_and_labels.md

```diff
@@ -19,6 +21,10 @@ This page defines common annotations and labels we set in Kubernetes objects.
 
 - `app` - should be set to the same value as `app.kubernetes.io/name`. It's
   deprecated and only kept to avoid breaking existing workflows.
+- `cluster.x-k8s.io/cluster-name` - currently set to same value as
```

Comment:
Is this only the equivalent to `giantswarm.io/cluster`? Will we put `cluster.x-k8s.io/cluster-name` also in all CRs associated with the cluster like we do with `giantswarm.io/cluster`?

Reply:
Yes, that would be the idea. We should also start widely using `OwnerReferences` correspondingly, though that's slightly different thing.

Reply:
Specifically we will put `cluster.x-k8s.io/cluster-name` to `MachinePool` and `AzureMachinePool` as well (and I believe `AzureMachine` used for the master nodes). I'll open another `fmt` PR for this (just to clarify first few other details around `MachinePool`/`AzureMachinePool` labels and annotations).